tests/ubpf_verifier.rs

```diff
@@ -16,11 +16,15 @@
 
 // These are unit tests for the eBPF “verifier”.
 
+// These tests should only be run on std as they depend on the assembler.
+#![cfg(feature = "std")]
```

Comment:
Not all of them do, but OK. I'd adjust the comment to say "most" or even "many of these tests", rather than implying they all do.

Reply:
Yes, you are right indeed. It was a mistake on my part, I saw that compilation failed because the module was trying to import the assembler so I just disabled the whole test file. Now it is fixed, the assembler gets included conditionally and only the tests that depend on it are disabled when testing `no_std`